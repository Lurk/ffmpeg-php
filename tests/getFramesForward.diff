001+ Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFramesForward.php on line 8
001- ffmpeg getFramesForward(1): md5 = 054ca5377a0592ea15497bcfefa59181
002- ffmpeg getFramesForward(2): md5 = 054ca5377a0592ea15497bcfefa59181
003- ffmpeg getFramesForward(3): md5 = 054ca5377a0592ea15497bcfefa59181
004- ffmpeg getFramesForward(4): md5 = bc3e96ad73e019583d449b6d2d857858
005- ffmpeg getFramesForward(5): md5 = bc3e96ad73e019583d449b6d2d857858
006- ffmpeg getFramesForward(6): md5 = bc3e96ad73e019583d449b6d2d857858
007- ffmpeg getFramesForward(7): md5 = 0a7a900dc376afc5598d5b2ae50be442
008- ffmpeg getFramesForward(8): md5 = 0a7a900dc376afc5598d5b2ae50be442
009- ffmpeg getFramesForward(9): md5 = 0a7a900dc376afc5598d5b2ae50be442
010- ffmpeg getFramesForward(10): md5 = 8de055e9eb38073ab35f75cfe4adae83
011- ffmpeg getFramesForward(11): md5 = 8de055e9eb38073ab35f75cfe4adae83
012- ffmpeg getFramesForward(12): md5 = 8de055e9eb38073ab35f75cfe4adae83
013- ffmpeg getFramesForward(13): md5 = 73327939f7fc8b6080525dc259bdd136
014- ffmpeg getFramesForward(14): md5 = 73327939f7fc8b6080525dc259bdd136
015- ffmpeg getFramesForward(15): md5 = 73327939f7fc8b6080525dc259bdd136
016- ffmpeg getFramesForward(16): md5 = 25fce65aaebf63d787415133ee525e0b
017- ffmpeg getFramesForward(17): md5 = 25fce65aaebf63d787415133ee525e0b
018- ffmpeg getFramesForward(18): md5 = 25fce65aaebf63d787415133ee525e0b
019- ffmpeg getFramesForward(19): md5 = dbcd63a3fe7bc6e5ae746134462a582d
020- ffmpeg getFramesForward(20): md5 = dbcd63a3fe7bc6e5ae746134462a582d
021- ffmpeg getFramesForward(21): md5 = dbcd63a3fe7bc6e5ae746134462a582d
022- ffmpeg getFramesForward(22): md5 = 5af7473006c39343accad86b6f9cf6ed
023- ffmpeg getFramesForward(23): md5 = 5af7473006c39343accad86b6f9cf6ed
024- ffmpeg getFramesForward(24): md5 = 5af7473006c39343accad86b6f9cf6ed
025- ffmpeg getFramesForward(25): md5 = 405cc6f1453352460cf01875aa23ccff
026- ffmpeg getFramesForward(26): md5 = 405cc6f1453352460cf01875aa23ccff
027- ffmpeg getFramesForward(27): md5 = 405cc6f1453352460cf01875aa23ccff
028- ffmpeg getFramesForward(28): md5 = cbdbcebeea6ea45f6965a83543b6c44b
029- ffmpeg getFramesForward(29): md5 = cbdbcebeea6ea45f6965a83543b6c44b
030- ffmpeg getFramesForward(30): md5 = cbdbcebeea6ea45f6965a83543b6c44b
031- ffmpeg getFramesForward(31): md5 = 9620a937f1a19c3d5e95716fae1eb47f
032- ffmpeg getFramesForward(32): md5 = 9620a937f1a19c3d5e95716fae1eb47f
033- ffmpeg getFramesForward(33): md5 = 9620a937f1a19c3d5e95716fae1eb47f
034- ffmpeg getFramesForward(34): md5 = 689ad71143a347f53417356eabb0037f
035- ffmpeg getFramesForward(35): md5 = 689ad71143a347f53417356eabb0037f
036- ffmpeg getFramesForward(36): md5 = 689ad71143a347f53417356eabb0037f
037- ffmpeg getFramesForward(37): md5 = 4f9a4e239c4b9459a59b038b0be3d5df
038- ffmpeg getFramesForward(38): md5 = 4f9a4e239c4b9459a59b038b0be3d5df
039- ffmpeg getFramesForward(39): md5 = 4f9a4e239c4b9459a59b038b0be3d5df
040- ffmpeg getFramesForward(40): md5 = 579618d56fe46acdb9bcfcff67808ae4
041- ffmpeg getFramesForward(41): md5 = 579618d56fe46acdb9bcfcff67808ae4
042- ffmpeg getFramesForward(42): md5 = 579618d56fe46acdb9bcfcff67808ae4
043- ffmpeg getFramesForward(43): md5 = 9f77876e4926b92d5f7dc565ac94e366
044- ffmpeg getFramesForward(44): md5 = 9f77876e4926b92d5f7dc565ac94e366
045- ffmpeg getFramesForward(45): md5 = 9f77876e4926b92d5f7dc565ac94e366
046- ffmpeg getFramesForward(46): md5 = faa5ef30ddeb97020250c150779e3399
047- ffmpeg getFramesForward(47): md5 = faa5ef30ddeb97020250c150779e3399
048- ffmpeg getFramesForward(48): md5 = faa5ef30ddeb97020250c150779e3399
049- ffmpeg getFramesForward(49): md5 = c61c4e4620af5adba309237a84ef91df
050- ffmpeg getFramesForward(50): md5 = c61c4e4620af5adba309237a84ef91df
051- ffmpeg getFramesForward(51): md5 = c61c4e4620af5adba309237a84ef91df
052- ffmpeg getFramesForward(52): md5 = 3953dcbdeb9b687b2b1e925183a16c39
053- ffmpeg getFramesForward(53): md5 = 3953dcbdeb9b687b2b1e925183a16c39
054- ffmpeg getFramesForward(54): md5 = 3953dcbdeb9b687b2b1e925183a16c39
055- ffmpeg getFramesForward(55): md5 = c360fd94742147db7e97796a61eee19f
056- ffmpeg getFramesForward(56): md5 = c360fd94742147db7e97796a61eee19f
057- ffmpeg getFramesForward(57): md5 = c360fd94742147db7e97796a61eee19f
058- ffmpeg getFramesForward(58): md5 = 7b1f32d4e4a1efce454c0b73ec015e75
059- ffmpeg getFramesForward(59): md5 = 7b1f32d4e4a1efce454c0b73ec015e75
060- ffmpeg getFramesForward(60): md5 = 7b1f32d4e4a1efce454c0b73ec015e75
061- ffmpeg getFramesForward(61): md5 = 963b85dafe7ac84daf64a12931509de7
062- ffmpeg getFramesForward(62): md5 = 963b85dafe7ac84daf64a12931509de7
063- ffmpeg getFramesForward(63): md5 = 963b85dafe7ac84daf64a12931509de7
064- ffmpeg getFramesForward(64): md5 = d86b04f95208b77c06e49ba9a76b477f
065- ffmpeg getFramesForward(65): md5 = d86b04f95208b77c06e49ba9a76b477f
066- ffmpeg getFramesForward(66): md5 = d86b04f95208b77c06e49ba9a76b477f
067- ffmpeg getFramesForward(67): md5 = cc52ca71fc8daae8feba52fa329f9f66
068- ffmpeg getFramesForward(68): md5 = cc52ca71fc8daae8feba52fa329f9f66
069- ffmpeg getFramesForward(69): md5 = cc52ca71fc8daae8feba52fa329f9f66
070- ffmpeg getFramesForward(70): md5 = 3cac4625a645a147a7e7cb8ab28ebf07
071- ffmpeg getFramesForward(71): md5 = 3cac4625a645a147a7e7cb8ab28ebf07
072- ffmpeg getFramesForward(72): md5 = 3cac4625a645a147a7e7cb8ab28ebf07
073- ffmpeg getFramesForward(73): md5 = f8eb6d231d353e8d98cf76f1fcf960c3
074- ffmpeg getFramesForward(74): md5 = f8eb6d231d353e8d98cf76f1fcf960c3
075- ffmpeg getFramesForward(75): md5 = f8eb6d231d353e8d98cf76f1fcf960c3
076- ffmpeg getFramesForward(76): md5 = 25ea525ec2630b123da3bd18b44e391b
077- ffmpeg getFramesForward(77): md5 = 25ea525ec2630b123da3bd18b44e391b
078- ffmpeg getFramesForward(78): md5 = 25ea525ec2630b123da3bd18b44e391b
079- ffmpeg getFramesForward(79): md5 = 97ea17ae200e6ca4e2412ba5546d4cdb
080- ffmpeg getFramesForward(80): md5 = 97ea17ae200e6ca4e2412ba5546d4cdb
081- ffmpeg getFramesForward(81): md5 = 97ea17ae200e6ca4e2412ba5546d4cdb
082- ffmpeg getFramesForward(82): md5 = d0bc5176b565500f09851abf059bf736
083- ffmpeg getFramesForward(83): md5 = d0bc5176b565500f09851abf059bf736
084- ffmpeg getFramesForward(84): md5 = d0bc5176b565500f09851abf059bf736
085- ffmpeg getFramesForward(85): md5 = a73c78e3ae667682e9920a218353540b
086- ffmpeg getFramesForward(86): md5 = a73c78e3ae667682e9920a218353540b
087- ffmpeg getFramesForward(87): md5 = a73c78e3ae667682e9920a218353540b
088- ffmpeg getFramesForward(88): md5 = 02172b5823992d273f7d4295f628309a
089- ffmpeg getFramesForward(89): md5 = 02172b5823992d273f7d4295f628309a
090- ffmpeg getFramesForward(90): md5 = 02172b5823992d273f7d4295f628309a
091- ffmpeg getFramesForward(91): md5 = 5887bfa75094567d9aec180eda7e45c6
092- ffmpeg getFramesForward(92): md5 = 5887bfa75094567d9aec180eda7e45c6
093- ffmpeg getFramesForward(93): md5 = 5887bfa75094567d9aec180eda7e45c6
094- ffmpeg getFramesForward(94): md5 = 3b93b082d52c4ccb8dc5beb22c8316ab
095- ffmpeg getFramesForward(95): md5 = 3b93b082d52c4ccb8dc5beb22c8316ab
096- ffmpeg getFramesForward(96): md5 = 3b93b082d52c4ccb8dc5beb22c8316ab
097- ffmpeg getFramesForward(97): md5 = b6485f3287312414e90fc0508ce9b194
098- ffmpeg getFramesForward(98): md5 = b6485f3287312414e90fc0508ce9b194
099- ffmpeg getFramesForward(99): md5 = b6485f3287312414e90fc0508ce9b194
100- ffmpeg getFramesForward(100): md5 = 57fe0edffdfcf06e88d7219662f04b05
101- ffmpeg getFramesForward(101): md5 = 57fe0edffdfcf06e88d7219662f04b05
102- ffmpeg getFramesForward(102): md5 = 57fe0edffdfcf06e88d7219662f04b05
103- ffmpeg getFramesForward(103): md5 = 5e6ab28f31801cddb176f1c01d13f364
104- ffmpeg getFramesForward(104): md5 = 5e6ab28f31801cddb176f1c01d13f364
105- ffmpeg getFramesForward(105): md5 = 5e6ab28f31801cddb176f1c01d13f364
106- ffmpeg getFramesForward(106): md5 = ab2142ee9ec11e01e42a072ad5762030
107- ffmpeg getFramesForward(107): md5 = ab2142ee9ec11e01e42a072ad5762030
108- ffmpeg getFramesForward(108): md5 = ab2142ee9ec11e01e42a072ad5762030
109- ffmpeg getFramesForward(109): md5 = 84410c79c7cf32562a83fa8f4bb981f7
110- ffmpeg getFramesForward(110): md5 = 84410c79c7cf32562a83fa8f4bb981f7
111- ffmpeg getFramesForward(111): md5 = 84410c79c7cf32562a83fa8f4bb981f7
112- ffmpeg getFramesForward(112): md5 = 5ea25236d676122a90799adefc529611
113- ffmpeg getFramesForward(113): md5 = 5ea25236d676122a90799adefc529611
114- ffmpeg getFramesForward(114): md5 = 5ea25236d676122a90799adefc529611
115- ffmpeg getFramesForward(115): md5 = 0ac65e51db3d7b40f55df3a43f3f03d2
116- ffmpeg getFramesForward(116): md5 = 0ac65e51db3d7b40f55df3a43f3f03d2
117- ffmpeg getFramesForward(117): md5 = 0ac65e51db3d7b40f55df3a43f3f03d2
118- ffmpeg getFramesForward(118): md5 = b0b6a709c28c75d9beffaaaba969ec12
119- ffmpeg getFramesForward(119): md5 = b0b6a709c28c75d9beffaaaba969ec12
120- ffmpeg getFramesForward(120): md5 = b0b6a709c28c75d9beffaaaba969ec12
121- ffmpeg getFramesForward(121): md5 = 7c58a3bc631a7ec1dd48de99658a5cc4
122- ffmpeg getFramesForward(122): md5 = 7c58a3bc631a7ec1dd48de99658a5cc4
123- ffmpeg getFramesForward(123): md5 = 7c58a3bc631a7ec1dd48de99658a5cc4
124- ffmpeg getFramesForward(124): md5 = a89b9a83054bf8737a97c080f07a59bd
125- ffmpeg getFramesForward(125): md5 = a89b9a83054bf8737a97c080f07a59bd
126- ffmpeg getFramesForward(126): md5 = a89b9a83054bf8737a97c080f07a59bd
127- ffmpeg getFramesForward(127): md5 = 5d58afe8118077391b8523a065dd7fce
128- ffmpeg getFramesForward(128): md5 = 5d58afe8118077391b8523a065dd7fce
129- ffmpeg getFramesForward(129): md5 = 5d58afe8118077391b8523a065dd7fce
130- ffmpeg getFramesForward(130): md5 = d7a36b7a67f4ca8941931a15517f5eb1
131- ffmpeg getFramesForward(131): md5 = d7a36b7a67f4ca8941931a15517f5eb1
132- ffmpeg getFramesForward(132): md5 = d7a36b7a67f4ca8941931a15517f5eb1
133- ffmpeg getFramesForward(133): md5 = 3064adc7a54286b4c45f9fe3669e8b1d
134- ffmpeg getFramesForward(134): md5 = 3064adc7a54286b4c45f9fe3669e8b1d
135- ffmpeg getFramesForward(135): md5 = 3064adc7a54286b4c45f9fe3669e8b1d
136- ffmpeg getFramesForward(136): md5 = 49feb765089580d9f3de8a6e36cbaed7
137- ffmpeg getFramesForward(137): md5 = 49feb765089580d9f3de8a6e36cbaed7
138- ffmpeg getFramesForward(138): md5 = 49feb765089580d9f3de8a6e36cbaed7
139- ffmpeg getFramesForward(139): md5 = 3f4495bb347480746f2fb994465b25a9
140- ffmpeg getFramesForward(140): md5 = 3f4495bb347480746f2fb994465b25a9
141- ffmpeg getFramesForward(141): md5 = 3f4495bb347480746f2fb994465b25a9
142- ffmpeg getFramesForward(142): md5 = 53343b23bb1a4aeb1f667608265fad9a
143- ffmpeg getFramesForward(143): md5 = 53343b23bb1a4aeb1f667608265fad9a
144- ffmpeg getFramesForward(144): md5 = 53343b23bb1a4aeb1f667608265fad9a
145- ffmpeg getFramesForward(145): md5 = b84c2d59f936f7cae83e736dc2dd0ebc
146- ffmpeg getFramesForward(146): md5 = b84c2d59f936f7cae83e736dc2dd0ebc
147- ffmpeg getFramesForward(147): md5 = b84c2d59f936f7cae83e736dc2dd0ebc
148- ffmpeg getFramesForward(148): md5 = abe4ae47b43678b60c192fbb4d0a69b5
149- ffmpeg getFramesForward(149): md5 = abe4ae47b43678b60c192fbb4d0a69b5
150- ffmpeg getFramesForward(150): md5 = abe4ae47b43678b60c192fbb4d0a69b5
151- ffmpeg getFramesForward(151): md5 = 117c46c3a208183085c41eccf223c7e6
152- ffmpeg getFramesForward(152): md5 = 117c46c3a208183085c41eccf223c7e6
153- ffmpeg getFramesForward(153): md5 = 117c46c3a208183085c41eccf223c7e6
154- ffmpeg getFramesForward(154): md5 = d587fae1b39f3a2ffae867f9652e1ed7
155- ffmpeg getFramesForward(155): md5 = d587fae1b39f3a2ffae867f9652e1ed7
156- ffmpeg getFramesForward(156): md5 = d587fae1b39f3a2ffae867f9652e1ed7
157- ffmpeg getFramesForward(157): md5 = db17bcfe68abe673bf3df8b8062714ab
158- ffmpeg getFramesForward(158): md5 = db17bcfe68abe673bf3df8b8062714ab
159- ffmpeg getFramesForward(159): md5 = db17bcfe68abe673bf3df8b8062714ab
160- ffmpeg getFramesForward(160): md5 = 316351119812a64ff994bbad9332286e
161- ffmpeg getFramesForward(161): md5 = 316351119812a64ff994bbad9332286e
162- ffmpeg getFramesForward(162): md5 = 316351119812a64ff994bbad9332286e
163- ffmpeg getFramesForward(163): md5 = f4af5e8779fa40f07b0285a4dc1b049d
164- ffmpeg getFramesForward(164): md5 = f4af5e8779fa40f07b0285a4dc1b049d
165- ffmpeg getFramesForward(165): md5 = f4af5e8779fa40f07b0285a4dc1b049d
166- ffmpeg getFramesForward(166): md5 = 274f9691f9d2360d53f299c1fba79614
167- ffmpeg getFramesForward(167): md5 = 274f9691f9d2360d53f299c1fba79614
168- ffmpeg getFramesForward(168): md5 = 274f9691f9d2360d53f299c1fba79614
169- ffmpeg getFramesForward(169): md5 = 4973cb1e537815ee8b7ef73e759f5c39
170- ffmpeg getFramesForward(170): md5 = 4973cb1e537815ee8b7ef73e759f5c39
171- ffmpeg getFramesForward(171): md5 = 4973cb1e537815ee8b7ef73e759f5c39
172- ffmpeg getFramesForward(172): md5 = 8629707f6d5c1a205564ac5f6f10d89f
173- ffmpeg getFramesForward(173): md5 = 8629707f6d5c1a205564ac5f6f10d89f
174- ffmpeg getFramesForward(174): md5 = 8629707f6d5c1a205564ac5f6f10d89f
175- ffmpeg getFramesForward(175): md5 = f72576077cd616c026c7f56829f220ae
176- ffmpeg getFramesForward(176): md5 = f72576077cd616c026c7f56829f220ae
177- ffmpeg getFramesForward(177): md5 = f72576077cd616c026c7f56829f220ae
178- ffmpeg getFramesForward(178): md5 = 017aaef2fe10b6aa2788d7ea0620c252
179- ffmpeg getFramesForward(179): md5 = 017aaef2fe10b6aa2788d7ea0620c252
180- ffmpeg getFramesForward(180): md5 = 017aaef2fe10b6aa2788d7ea0620c252
181- ffmpeg getFramesForward(181): md5 = a5b5bfe8b20e018ef506d703b2af7ccc
182- ffmpeg getFramesForward(182): md5 = a5b5bfe8b20e018ef506d703b2af7ccc
183- ffmpeg getFramesForward(183): md5 = a5b5bfe8b20e018ef506d703b2af7ccc
184- ffmpeg getFramesForward(184): md5 = cc781178ac22f4cf6641183926067215
185- ffmpeg getFramesForward(185): md5 = cc781178ac22f4cf6641183926067215
186- ffmpeg getFramesForward(186): md5 = cc781178ac22f4cf6641183926067215
187- ffmpeg getFramesForward(187): md5 = b0338a88dff4cbabb1599a807e5c6ef0
188- ffmpeg getFramesForward(188): md5 = b0338a88dff4cbabb1599a807e5c6ef0
189- ffmpeg getFramesForward(189): md5 = b0338a88dff4cbabb1599a807e5c6ef0
190- ffmpeg getFramesForward(190): md5 = d1ee4ba4a11217efd0cf8cba15a2eb1f
191- ffmpeg getFramesForward(191): md5 = d1ee4ba4a11217efd0cf8cba15a2eb1f
192- ffmpeg getFramesForward(192): md5 = d1ee4ba4a11217efd0cf8cba15a2eb1f
193- ffmpeg getFramesForward(193): md5 = 0d2849f432fc9656c31629f6288feba0
194- ffmpeg getFramesForward(194): md5 = 0d2849f432fc9656c31629f6288feba0
195- ffmpeg getFramesForward(195): md5 = 0d2849f432fc9656c31629f6288feba0
196- ffmpeg getFramesForward(196): md5 = c26f6d4cf0737c5c6b102c048312f0fc
197- ffmpeg getFramesForward(197): md5 = c26f6d4cf0737c5c6b102c048312f0fc
198- ffmpeg getFramesForward(198): md5 = c26f6d4cf0737c5c6b102c048312f0fc
199- ffmpeg getFramesForward(199): md5 = 23d6d68503ec607327ee6216bc695593
200- ffmpeg getFramesForward(200): md5 = 23d6d68503ec607327ee6216bc695593
201- ffmpeg getFramesForward(201): md5 = 23d6d68503ec607327ee6216bc695593
202- ffmpeg getFramesForward(202): md5 = 85c3d4dbcd9315b00e220d334a012083
203- ffmpeg getFramesForward(203): md5 = 85c3d4dbcd9315b00e220d334a012083
204- ffmpeg getFramesForward(204): md5 = 85c3d4dbcd9315b00e220d334a012083
205- ffmpeg getFramesForward(205): md5 = 08d8b9d438667d710cacc0d9081a4883
206- ffmpeg getFramesForward(206): md5 = 08d8b9d438667d710cacc0d9081a4883
207- ffmpeg getFramesForward(207): md5 = 08d8b9d438667d710cacc0d9081a4883
208- ffmpeg getFramesForward(208): md5 = d3ce913a9c16206990a6b9a80c0561f3
209- ffmpeg getFramesForward(209): md5 = d3ce913a9c16206990a6b9a80c0561f3
210- ffmpeg getFramesForward(210): md5 = d3ce913a9c16206990a6b9a80c0561f3
211- ffmpeg getFramesForward(211): md5 = 64bcd46af5519ad65722bc94e71a5628
212- ffmpeg getFramesForward(212): md5 = 64bcd46af5519ad65722bc94e71a5628
213- ffmpeg getFramesForward(213): md5 = 64bcd46af5519ad65722bc94e71a5628
214- ffmpeg getFramesForward(214): md5 = 55a1b21fb233ec1b965aa27fc5680ef7
215- ffmpeg getFramesForward(215): md5 = 55a1b21fb233ec1b965aa27fc5680ef7
216- ffmpeg getFramesForward(216): md5 = 55a1b21fb233ec1b965aa27fc5680ef7
217- ffmpeg getFramesForward(217): md5 = cbf35af572f3947b04b391dbe8f01912
218- ffmpeg getFramesForward(218): md5 = cbf35af572f3947b04b391dbe8f01912
219- ffmpeg getFramesForward(219): md5 = cbf35af572f3947b04b391dbe8f01912
220- ffmpeg getFramesForward(220): md5 = b68f724605514e4ae7676c04fff62461
221- ffmpeg getFramesForward(221): md5 = b68f724605514e4ae7676c04fff62461
222- ffmpeg getFramesForward(222): md5 = b68f724605514e4ae7676c04fff62461
223- ffmpeg getFramesForward(223): md5 = e2d66cb30e1d942da4b603ecae95c64e
224- ffmpeg getFramesForward(224): md5 = e2d66cb30e1d942da4b603ecae95c64e
225- ffmpeg getFramesForward(225): md5 = e2d66cb30e1d942da4b603ecae95c64e
226- ffmpeg getFramesForward(226): md5 = 70c86d74b825b685d3e90c2dfa21f307
227- ffmpeg getFramesForward(227): md5 = 70c86d74b825b685d3e90c2dfa21f307
228- ffmpeg getFramesForward(228): md5 = 70c86d74b825b685d3e90c2dfa21f307
229- ffmpeg getFramesForward(229): md5 = 52e93862544f05c92375d2aaa2c7dc16
230- ffmpeg getFramesForward(230): md5 = 52e93862544f05c92375d2aaa2c7dc16
231- ffmpeg getFramesForward(231): md5 = 52e93862544f05c92375d2aaa2c7dc16
232- ffmpeg getFramesForward(232): md5 = da9910064c0414e256056b769f942049
233- ffmpeg getFramesForward(233): md5 = da9910064c0414e256056b769f942049
234- ffmpeg getFramesForward(234): md5 = da9910064c0414e256056b769f942049
235- ffmpeg getFramesForward(235): md5 = 2817e435640fc7c26685401007ad6177
236- ffmpeg getFramesForward(236): md5 = 2817e435640fc7c26685401007ad6177
237- ffmpeg getFramesForward(237): md5 = 2817e435640fc7c26685401007ad6177
238- ffmpeg getFramesForward(238): md5 = 5da7388e5b099cfb030b36cefcb260aa
239- ffmpeg getFramesForward(239): md5 = 5da7388e5b099cfb030b36cefcb260aa
240- ffmpeg getFramesForward(240): md5 = 5da7388e5b099cfb030b36cefcb260aa