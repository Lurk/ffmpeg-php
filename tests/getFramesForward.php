<?php
$mov = new ffmpeg_movie(dirname(__FILE__) . '/test_media/robot.avi');

$framecount = $mov->getFrameCount();
for($i = 1; $i <= $framecount; $i++) {
    $img = sprintf("%s/test-%04d.png", dirname(__FILE__), $i);
    $frame = $mov->getFrame($i);
    $image = $frame->toGDImage();
    imagepng($image, $img);
    printf("ffmpeg getFramesForward($i): md5 = %s\n", md5(file_get_contents($img)));
    imagedestroy($image);
    unlink($img);
}
?>
