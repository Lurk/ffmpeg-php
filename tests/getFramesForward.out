Fatal error: failed to convert frame to gd image in /src/ffmpeg-php-code/ffmpeg-php/tests/getFramesForward.php on line 8